tools/dep_updaters/update-icu.sh

```diff
@@ -84,6 +82,7 @@ rm -rf "$DEPS_DIR/icu"
 
 perl -i -pe "s|\"url\": .*|\"url\": \"$NEW_VERSION_TGZ_URL\",|" "$TOOLS_DIR/icu/current_ver.dep"
 
+perl -i -pe "$REGEX" "$TOOLS_DIR/icu/current_ver.dep"
```

Comment:
conflict resolved 
and changed writing of current_ver.dep to be done after icu file change